pyproject.toml

```
@@ -0,0 +1,39 @@
+[project]
+name = "taskgraph"
+description = "Parallel task graph framework"
+readme = "README.rst"
+requires-python = ">=3.6"
+license = {file = "LICENSE.txt"}
+maintainers = [
+    {name = "Natural Capital Project Software Team"}
+]
+keywords = ["parallel", "multiprocessing", "distributed", "computing"]
+classifiers = [
+    "Intended Audience :: Developers",
+    "Topic :: System :: Distributed Computing",
+    "Development Status :: 5 - Production/Stable",
+    "Natural Language :: English",
+    "Operating System :: MacOS :: MacOS X",
+    "Operating System :: Microsoft",
+    "Operating System :: POSIX",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.11",
```

haha, I'm not sure what happened but this got flipped to 3.11! 
```suggestion
    "Programming Language :: Python :: 3.12",
```